src/main/java/mate/controller/car/DeleteCarController.java

```diff
@@ -0,0 +1,23 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/delete")
+public class DeleteCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        carService.delete(id);
+    }
```

Comment:
After deleting a car, it is good practice to redirect the user to a relevant page (e.g., the list of all cars) to confirm the action was successful.